src/tests/api/HappyPath.test.ts

```diff
@@ -0,0 +1,244 @@
+import { sessionPost, stubStartPost, postDocumentSelection, startStubServiceAndReturnSessionId, getSessionById, getSessionAndVerifyKey, authorizationGet, tokenPost, userInfoPost, sessionConfigurationGet, postAbortSession } from "./ApiTestSteps";
+import { getTxmaEventsFromTestHarness, validateTxMAEventData } from "./ApiUtils";
+import f2fStubPayload from "../data/exampleStubPayload.json";
+import thinFilePayload from "../data/thinFilePayload.json";
+import exampleStubPayload from "../data/exampleStubPayload.json";
```

Comment:
We have the same import twice but names different things? Could we just change this to one import?

Reply:
Updated so we only have 1 import